packages/common/src/config.ts

```diff
@@ -371,8 +381,65 @@ export const EmbedChainInfos: AppChainInfo[] = [
     features: ['stargate', 'ibc-transfer', 'cosmwasm']
   },
   {
-    chainId: 'oraibtc-subnet-1',
+    chainId: 'oraibtc-mainnet-1',
     chainName: 'OraiBtc Bridge',
+    rpc: 'https://btc.rpc.orai.io',
+    rest: 'https://btc.lcd.orai.io',
+    networkType: 'cosmos',
+    stakeCurrency: {
+      coinDenom: 'ORAIBTC',
+      coinMinimalDenom: 'uoraibtc',
+      coinDecimals: 6,
+      gasPriceStep: {
+        low: 0,
+        average: 0,
+        high: 0
+      },
+      coinImageUrl: 'https://assets.coingecko.com/coins/images/1/small/bitcoin.png'
+    },
+    bip44: {
+      coinType: 118
+    },
+    coinType: 118,
+    bech32Config: Bech32Address.defaultBech32Config('oraibtc'),
+    // List of all coin/tokens used in this chain.
+    get currencies() {
+      return [
+        this.stakeCurrency,
+        {
+          coinDenom: 'oBTC',
+          coinMinimalDenom: 'usat',
+          coinDecimals: 14,
+          gasPriceStep: {
+            low: 0,
+            average: 0,
+            high: 0
+          },
+          coinImageUrl: 'https://assets.coingecko.com/coins/images/1/small/bitcoin.png'
+        }
+      ];
+    },
+    get feeCurrencies() {
+      return [
+        this.stakeCurrency,
+        {
+          coinDenom: 'oBTC',
+          coinMinimalDenom: 'usat',
+          coinDecimals: 14,
+          gasPriceStep: {
+            low: 0,
+            average: 0,
+            high: 0
+          },
+          coinImageUrl: 'https://assets.coingecko.com/coins/images/1/small/bitcoin.png'
+        }
+      ];
+    },
+    features: ['stargate', 'ibc-transfer', 'cosmwasm']
+  },
+  {
+    chainId: 'oraibtc-subnet-1',
+    chainName: 'OraiBtc Bridge Testnet',
```

Comment:
The chain name should be OraiBtc Bridge :D

Reply:
this is a network test for oraibtc bridge